package.json

```diff
@@ -1,22 +1,43 @@
 {
   "name": "stellar-base",
   "version": "8.2.2",
-  "description": "Low level stellar support library",
+  "description": "Low-level support library for the Stellar network.",
   "main": "./lib/index.js",
   "types": "./types/index.d.ts",
+  "type": "module",
   "scripts": {
-    "test": "gulp test:node",
-    "test:watch": "gulp test:watch",
-    "docs": "jsdoc -c .jsdoc.json --verbose",
-    "dtslint": "dtslint types --localTs node_modules/typescript/lib",
-    "preversion": "gulp test",
-    "version": "gulp build",
-    "postversion": "git push && git push --tags",
-    "prettier-all": "prettier --write **/*.js"
+    "build": "yarn build:node",
+    "build:node": "babel --out-dir lib/ src/",
```

Comment:
also `build:node:prod` for completeness?

Reply:
for completeness, true, but there is no prod setting for the node side (for the browser it does a minify and no source map)